src/pmpo_mesh.cpp

```diff
@@ -48,9 +48,10 @@ namespace polyMPO{
         
         auto elmMassMapEntry = meshFields2TypeAndString.at(MeshF_ElmMass);
         PMT_ALWAYS_ASSERT(elmMassMapEntry.first == MeshFType_ElmBased);
-        elmMass_ = MeshFView<MeshF_ElmMass>(elmMassMapEntry.second,numElms_);
+        elmMass_      = MeshFView<MeshF_ElmMass>(elmMassMapEntry.second,numElms_);
+        elmCenterXYZ_ = MeshFView<MeshF_ElmCenterXYZ>(meshFields2TypeAndString.at(MeshF_ElmCenterXYZ).second, numElms_);
```

Comment:
may be remove space after comma: `second,numElms_`, like previous line

Reply:
Done